### DISCUSSION TITLE
Give access for individual documents

### DISCUSSION THREAD

**Discussion by NangeGao** [outline/outline#5738] [Ideas]
Now I can give the access permissions of the entire collection to others. But sometimes I wanna set the access permissions for a single document in the collection to others. 
Could you please add a new feature for this?

**Comment by alkanna** [comment]
This is being built here FYI : https://github.com/outline/outline/pull/5814

**Reply by NangeGao** [reply]
I see. Thank you. But it has not merged to any release version yet?

**Reply by Limezy** [reply]
Not yet !
Will likely be part of the v0.75 release
